pkgs/top-level/all-packages.nix

```
@@ -5590,6 +5590,8 @@ in
 
   nwdiag = with python3Packages; toPythonApplication nwdiag;
 
+  nxpmicro-mfgtools = callPackage ../applications/science/electronics/nxpmicro-mfgtools { };
```

```suggestion
  nxpmicro-mfgtools = callPackage ../applications/science/electronics/nxpmicro-mfgtools { };
```

Many other derive flashing utilities (e.g. `esptool`, `stm32flash`) are in `development/tools/misc`, so that may be the more logical location.